Dockerfile

```diff
@@ -1,4 +1,34 @@
-FROM node:12 AS bootstrap
+FROM node:12
+
+ARG COMMIT_ID
+ENV COMMIT_ID=$COMMIT_ID
+
+ARG PANOPTES_ENV=production
+ENV PANOPTES_ENV=$PANOPTES_ENV
+
+ARG NODE_ENV=production
+ENV NODE_ENV=$NODE_ENV
+
+ARG APP_ENV=production
+ENV APP_ENV=$APP_ENV
+
+ENV NEXT_TELEMETRY_DISABLED=1
+
+ARG CONTENTFUL_ACCESS_TOKEN
+
+ARG CONTENTFUL_SPACE_ID
```

Comment:
Should these arguments also set env vars? They don't in the current setup but I don't quite follow how the Contentful secrets are being passed to the `app-content-pages` build environment.
https://github.com/zooniverse/front-end-monorepo/blob/c0e233b955cf37948566ae92fe19fa6842b71e8a/packages/app-content-pages/Dockerfile#L9-L12

Reply:
I've run a test build in Jenkins, without pushing the built image, and this seems to work fine as it is.
https://jenkins.zooniverse.org/job/Zooniverse%20GitHub/job/front-end-monorepo/job/PR-1887/24/

Reply:
I've also updated the Jenkinsfile so that we run a test build as part of CI.

Reply:
Jenkins is going to be slower now but that does mean we'll try a build of the static pages before changes get merged to staging.